rubin_sim/scheduler/detailers/short_survey.py

```diff
@@ -18,10 +18,15 @@ class Short_expt_detailer(Base_detailer):
         The short exposure time to use.
     nobs : float (2)
         The number of observations to try and take per year
+    night_max : float (None)
+        Do not apply any changes to the observation list if the current night is greater than night_max.
+        XXX--note, in theory, detailers should probably be expanded to include their own basis functions.
+    in_a_row : int (1)
+        How many short observations to do in a row.
```

Comment:
Maybe call this 'sequential' or 'sequence'? 
I guess the cadence notes and white papers on short exposures don't specify whether the short exposures have to be at the same or different times. Might try shooting Will a note to check into if they have any preference. Agreed it's probably more efficient to do sequential if there are more than one short exposure to be done overall.